kernel/kernel-defaults/src/test/scala/io/delta/kernel/defaults/ChecksumLogReplayMetricsTestBase.scala

```diff
@@ -145,4 +145,62 @@ trait ChecksumLogReplayMetricsTestBase extends LogReplayBaseSuite {
         readVersion = 10)
     }
   }
+
```

Comment:
Do we have a test if the CRC is present but no DM in it? Sorry I feel deja-vu asking this not sure if we've already covered this

Reply:
yeah, https://github.com/delta-io/delta/pull/4502/files#diff-91538c34b323c5cd743faf8b50c422a624ba134c0b7d1ca16a1ea60750c2a7f3L93-R87 is the test for the case.